### ISSUE TITLE
add quadratic weighted kappa loss function for multi-class classification

### ISSUE THREAD

**Issue by fernandezmartineznico** [orchardbirds/bokbokbok#75]
Add quadratic weighted kappa loss function for multi-class classification problems with categorical ordinal output.

Potential code implementation: https://www.kaggle.com/chenglongchen/customized-softkappa-loss-in-xgboost

**Comment by orchardbirds** [comment]
Thanks for the issue! 

I will take a look at it this week, unless you want to pick it up?

**Comment by orchardbirds** [comment]
@fernandezmartineznico This is now implemented as an eval metric for multi-class functions
